jupyter-js-widgets/css/widgets-base.css

```diff
@@ -93,6 +95,7 @@
 /* General Button Styling */
 
 .jupyter-button {
+    outline: none !important;
```

Comment:
This makes it impossible to see when the button is focused, which is a huge accessibility problem. Can we add some styling change for when the keyboard has focused the button?